docs/support/unbricking/unbrick-suzyq.md

```diff
@@ -60,7 +60,7 @@ Let's get to it:
        * `wget https://mrchromebox.tech/files/firmware/shellball/shellball.edgar.bin`
 
 ::: tip
-If you're not sure which file to use for your device / don't know your device's board name, you can reference [the supported devices page](supported-device.html).
+If you're not sure which file to use for your device / don't know your device's board name, you can reference [the supported devices page](https://docs.mrchromebox.tech/docs/supported-devices.html).
```

Comment:
we don't want to use the full URL here, only the relative one. `/docs/supported-devices.html` is the correct link